### PR TITLE
added tools for sideloading

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit e492d6faf218a48536f8553242e147ff38f43f21: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        8 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -334,33 +334,90 @@ Save the IPA file locally with the following command:
 
 #### Installing Apps
 
-When installing apps that are not available via the official distribution channel through Apple's App Store, this is called sideloading. There are various ways of sideloading which are described below.
+When installing apps by not using the official distribution channel through Apple's App Store, this is called sideloading. There are various ways of sideloading which are described below. On the iOS device, the actual installation process is then handled by the installd daemon, which will unpack and install the application. To integrate app services or be installed on an iOS device, all applications must be signed with a certificate issued by Apple. This means that the application can be installed only after successful code signature verification. On a jailbroken phone, however, you can circumvent this security feature with [AppSync](http://repo.hackyouriphone.org/appsyncunified), a package available in the Cydia store. It contains numerous useful applications that leverage jailbreak-provided root privileges to execute advanced functionality. AppSync is a tweak that patches installd, allowing the installation of fake-signed IPA packages.
```
Comment (TheDauntless) [inline Document/0x06b-Basic-Security-Testing.md]: > When installing apps by not using the official distribution channel through Apple's App Store,

to

> When you install an application without using Apple's App Store ?


+ Add the ALT text for the url

File: Document/0x06b-Basic-Security-Testing.md [hunk 2]
```diff
@@ -334,33 +334,90 @@ Save the IPA file locally with the following command:
...
 
 ##### libimobiledevice
 
-On Linux and also macOS, you can alternatively use [libimobiledevice](https://www.libimobiledevice.org/ "libimobiledevice"), a cross-platform software protocol library and a set of tools for native communication with iOS devices. This allows you to install apps over an USB connection via ideviceinstaller. The connection is implemented with the USB multiplexing daemon [usbmuxd](https://www.theiphonewiki.com/wiki/Usbmux "Usbmux"), which provides a TCP tunnel over USB.
+On Linux and also macOS, you can alternatively use [libimobiledevice](https://www.libimobiledevice.org/ "libimobiledevice"), a cross-platform software protocol library and a set of tools for native communication with iOS devices. This allows you to install apps over an USB connection by executing ideviceinstaller. The connection is implemented with the USB multiplexing daemon [usbmuxd](https://www.theiphonewiki.com/wiki/Usbmux "Usbmux"), which provides a TCP tunnel over USB.
```
Comment (TheDauntless) [inline Document/0x06b-Basic-Security-Testing.md]: an USB -> a USB

File: Document/0x06b-Basic-Security-Testing.md [hunk 3]
```diff
@@ -334,33 +334,90 @@ Save the IPA file locally with the following command:
...
 
 The package for libimobiledevice will be available in your Linux package manager. On macOS you can install libimobiledevice via brew:
 
 ```bash
 $ brew install libimobiledevice
 ```
 
-On the iOS device, the actual installation process is then handled by the installd daemon, which will unpack and install the application. To integrate app services or be installed on an iOS device, all applications must be signed with a certificate issued by Apple. This means that the application can be installed only after successful code signature verification. On a jailbroken phone, however, you can circumvent this security feature with [AppSync](http://repo.hackyouriphone.org/appsyncunified), a package available in the Cydia store. Cydia is an alternative app store or software distribution system. It contains numerous useful applications that leverage jailbreak-provided root privileges to execute advanced functionality. AppSync is a tweak that patches installd, allowing the installation of fake-signed IPA packages.
+After the installation you have several new command line tools available, like `ideviceinfo`, `ideviceinstaller` or `idevicedebug` to name a few.
```
Comment (TheDauntless) [inline Document/0x06b-Basic-Security-Testing.md]: like -> such as

And I would remove the "to name a few"

File: Document/0x06b-Basic-Security-Testing.md [hunk 4]
```diff
@@ -334,33 +334,90 @@ Save the IPA file locally with the following command:
...
+```
+
+After the installation, go into the directory of the IPA you want to install and unzip it as ios-deploy installs an app by using the bundle.
+
+```bash
+$ unzip Name.ipa
+$ ios-deploy --bundle 'Payload/Name.app' -W -d -v
+```
+
+After the app is installed on the iOS device, you can simply start it by adding the `-m` flag which will directly start debugging without app install.
```
Comment (TheDauntless) [inline Document/0x06b-Basic-Security-Testing.md]: without app install -> without installing the application again.

File: Document/0x06b-Basic-Security-Testing.md (hunk 1 above)
Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: done

File: Document/0x06b-Basic-Security-Testing.md (hunk 2 above)
Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: done

File: Document/0x06b-Basic-Security-Testing.md (hunk 3 above)
Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: done

File: Document/0x06b-Basic-Security-Testing.md (hunk 4 above)
Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: done

**Comment by OMTGreleaser** [comment]
Results for commit 59d3e99de59fc80e28e3984a8f266b93d4f6f954: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        8 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**